StonehearthEditor/pages/styles/main.css

```diff
@@ -112,23 +158,56 @@ button {
 .curve-table {
```

Comment:
this class is empty, maybe we don't need it.

Reply:
removed